code/modules/clothing/suits/hunting.dm

```diff
@@ -0,0 +1,57 @@
+/obj/item/clothing/suit/leather
+	name = "leather suit"
+	desc = "Crafted from fresh leather, may not stop a bullet or a knife, but it's a template to build off of."
+	icon_state = "suit_leather"
+	item_state = "suit_leather"
+	allowed = list(/obj/item/weapon/gun/energy, /obj/item/weapon/gun/projectile, /obj/item/ammo_storage,/obj/item/ammo_casing,/obj/item/weapon/melee/baton,/obj/item/weapon/handcuffs)
+	body_parts_covered = FULL_TORSO
+	heat_conductivity = SNOWGEAR_HEAT_CONDUCTIVITY
+	siemens_coefficient = 0.6
+	armor = list(melee = 25, bullet = 20, laser = 20, energy = 15, bomb = 20, bio = 0, rad = 0)
+
+/obj/item/clothing/suit/leather/attackby(obj/W, mob/user)
+	if(istype (W, /obj/item/stack/sheet/animalhide))
+		var/obj/item/stack/sheet/animalhide/S = W
+		if(S.amount < 2)
+			return ..() // ?
```

Comment:
Taken from girders.dm, as I was looking for some code that used stacks

else if(istype(W, /obj/item/stack/sheet))
		var/obj/item/stack/sheet/S = W
		switch(S.type)
			if(/obj/item/stack/sheet/metal, /obj/item/stack/sheet/metal/cyborg)
				if(state) //We are trying to finish a reinforced girder with regular metal
					return
				if(!anchored)
					if(S.amount < 2)
						return
					var/pdiff = performWallPressureCheck(src.loc)
					if(!pdiff) //Should really not be that precise, 10 kPa is the usual breaking point
						S.use(2)
						user.visible_message("<span class='warning'>[user] creates a false wall!</span>", \
						"<span class='notice'>You create a false wall. Push on it to open or close the passage.</span>")
						var/obj/structure/falsewall/FW = new /obj/structure/falsewall (src.loc)
						FW.add_hiddenprint(user)
						FW.add_fingerprint(user)
						qdel(src)
					else
						to_chat(user, "<span class='warning'>There is too much air moving through the gap!  The door wouldn't stay closed if you built it.</span>")
						message_admins("Attempted false wall made by [user.real_name] ([formatPlayerPanel(user,user.ckey)]) at [formatJumpTo(loc)] had a pressure difference of [pdiff]!")
						log_admin("Attempted false wall made by [user.real_name] (user.ckey) at [loc] had a pressure difference of [pdiff]!")
						return
				else
					if(S.amount < 2)
						return ..() // ?
					user.visible_message("<span class='notice'>[user] starts installing plating to \the [src].</span>", \
					"<span class='notice'>You start installing plating to \the [src].</span>")
					if(do_after(user, src, 40))
						if(S.amount < 2) //User being tricky
							return
						S.use(2)